classes/02class/exercises/c02-network04/raghunadhpokkalath/README.MD

```diff
@@ -0,0 +1,318 @@
+# c02-network04
+
+## Commands Execution Output
+
+- Commands for creating the subnets:
+```
+Availability Zone -  ap-southeast-2a
+
+private subnet [CIDR - 10.0.1.0/24]
+> aws ec2 create-subnet --vpc-id vpc-076a05ec0f3be4895  --availability-zone ap-southeast-2a --cidr-block 10.0.1.0/24 
```

Comment:
You can also set tags when creating by using the parameter `--tag-specifications 'ResourceType=subnet,Tags=[{Key=Name,Value=private-a}]'`